### PR TITLE
fix: check for ext ACK TLVs

### PR DESCRIPTION
**Pull request by adrianchiris**
checking for Extended ACK in nlmsghdr flags
should be done with bitwise AND op instead of OR op

### CODE REVIEW

File: nl/nl_linux.go [hunk 1]
```diff
@@ -661,12 +661,14 @@ func GetNetlinkSocketAt(newNs, curNs netns.NsHandle, protocol int) (*NetlinkSock
...
 // at the end of the code that needs to be executed in the network namespace.
 // Example:
-// func jobAt(...) error {
-//      d, err := executeInNetns(...)
-//      if err != nil { return err}
-//      defer d()
-//      < code which needs to be executed in specific netns>
-//  }
+//
+//	func jobAt(...) error {
```
Comment (adrianchiris) [inline nl/nl_linux.go:665]: this change is unrelated to the fix but the linter keeps adjusting the style.

**Comment by aboch** [comment]
Thanks for catching this one

**Comment by aboch** [comment]
LGTM

**Comment by aboch** [comment]
LGTM

**Pull request merged**